### PR TITLE
update to latest RAI vision and text environments

### PR DESCRIPTION
**Pull request by imatiach-msft**
Update the RAI Vision and Text environments to latest.
This PR precedes the PR to update the components to latest code.

### CODE REVIEW

File: responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml [hunk 1]
```diff
@@ -5,31 +5,22 @@ channels:
   - anaconda
   - pytorch
 dependencies:
-  - python=3.8
+  - python=3.7
```
Comment (vinuthakaranth) [inline responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:8]: why are we reducing python to 3.7? also yaml file has py138 in its name

File: responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml [hunk 2]
```diff
@@ -5,31 +5,22 @@ channels:
...
     - markupsafe<2.1.0
-    # Workaround to fix import failure related to itsdangerous's latest release: https://stackoverflow.com/questions/71189819/python-docker-importerror-cannot-import-name-json-from-itsdangerous
     - itsdangerous==2.0.1
-    - azureml-dataset-runtime
-    - azureml-core
-    - azureml-mlflow
-    - https://publictestdatasets.blob.core.windows.net/packages/pypi/responsibleai-vision/responsibleai_vision-0.0.8-py3-none-any.whl
-    - https://publictestdatasets.blob.core.windows.net/packages/pypi/azureml-model-serializer/azureml_model_serializer-0.0.2-py3-none-any.whl
+    - https://publictestdatasets.blob.core.windows.net/packages/pypi/responsibleai-vision/responsibleai_vision-0.0.15-py3-none-any.whl
+    - opencv-python==4.3.0.36
```
Comment (vinuthakaranth) [inline responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:23]: looks like we are pinning versions to many packages. Why are we pinning these versions, is there any issue we are trying to resolve?

**Review by vinuthakaranth — APPROVED** [review]

File: responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml (hunk 2 above)
Comment (imatiach-msft) [reply responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:23]: due to automl dependency

File: responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml (hunk 1 above)
Comment (imatiach-msft) [reply responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:8]: due to automl dependency

**Pull request merged**